src/h_vialib/secure/url.py

```diff
@@ -117,15 +117,21 @@ def _add_token(self, url, token):
 class ViaSecureURL(SecureURL):
     """A token for signing proxied URLs."""
 
-    MAX_AGE = timedelta(hours=1)
+    MAX_AGE = timedelta(hours=25)
```

Comment:
I think we should probably keep this to 1 hour by default and use the `max_age` value you added below to control the age when we sign Google URLs.

Reply:
Removed